CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [1.12.0] - 2021-04-29
```

Comment:
Remove this line with the version and date. When you releasy a new version this will be updated as well.

Reply:
Done. Thanks!